### ISSUE TITLE
ruby-grapghql 2.0 compatibility?

### ISSUE THREAD

**Issue by maltesa** [yabeda-rb/yabeda-graphql#3]
When I add the yabeda-graphql gem to my Gemfile, I receive the following error:

```bash
Bundler could not find compatible versions for gem "graphql":
  In snapshot (Gemfile.lock):
    graphql (= 2.0.4)

  In Gemfile:
    graphql (~> 2.0.4)

    yabeda-graphql was resolved to 0.2.1, which depends on
      graphql (~> 1.9)
```

Is it done with a change to the gemspec or is yabeda-graphql not yet compatible with `ruby-graphql ~> 2.x.x`?

**Comment by Envek** [comment]
Hey! Thanks for the heads up.

Allowed graphql gem 2.x in the gemspec and tests are passing, so released [version 0.2.2](https://github.com/yabeda-rb/yabeda-graphql/releases/tag/v0.2.2), enjoy!

P.S> OMG! SOMEONE IS GOING TO USE IT! :excited:

**Comment by maltesa** [comment]
Haha 👍🏼 Thanks for the immediate help! I'm definitely going to use it 😊
